ci-operator/step-registry/idp/htpasswd/idp-htpasswd-commands.sh

```diff
@@ -4,6 +4,48 @@ set -e
 set -u
 set -o pipefail
 
+function check_if_hypershift_env () {
+    if [ -f "${SHARED_DIR}/nested_kubeconfig" ]; then
+        IS_HYPERSHIFT_ENV="yes"
+    else
+        IS_HYPERSHIFT_ENV="no"
+        return 0
+    fi
+    MC_KUBECONFIG_FILE="${SHARED_DIR}/hs-mc.kubeconfig"
+    if [ -f "${MC_KUBECONFIG_FILE}" ]; then
+        export KUBECONFIG="${SHARED_DIR}/hs-mc.kubeconfig"
+        _jsonpath="{.items[?(@.metadata.name==\"$(cat ${SHARED_DIR}/cluster-name)\")].metadata.namespace}"
+        HYPERSHIFT_NAMESPACE=$(oc get hostedclusters -A -ojsonpath="$_jsonpath")
+    elif [ -f "${SHARED_DIR}/mgmt_kubeconfig" ]; then
+        export KUBECONFIG="${SHARED_DIR}/mgmt_kubeconfig"
+    else
+        export KUBECONFIG="${SHARED_DIR}/kubeconfig"
+    fi
+    
+    count=$(oc get hostedclusters --no-headers --ignore-not-found -n "$HYPERSHIFT_NAMESPACE" | wc -l)
```

Comment:
@xingxingxia Look like need to fix the namespace issue, [causing the pipeline failure](https://gcsweb-qe-private-deck-ci.apps.ci.l2s4.p1.openshiftapps.com/gcs/qe-private-deck/pr-logs/pull/openshift_release/53677/rehearse-53677-periodic-ci-openshift-openshift-tests-private-release-4.16-amd64-nightly-aws-ipi-ovn-hypershift-fips-guest-f14/1806271426114621440/artifacts/aws-ipi-ovn-hypershift-fips-guest-f14/idp-htpasswd/build-log.txt). 

```
no proxy setting.
hostedcluster count: 1
runtime_env does not exist, continuing checking...
Error from server (NotFound): namespaces "ci-op-vrgbdlz9" not found

```